meddlr/data/transforms/2D_motion.py

```diff
@@ -0,0 +1,87 @@
+from typing import Optional, Tuple, Sequence, Union
+
+import math
+import numpy as np
+import torch
+import meddlr.ops as F
+from meddlr.transforms import RandomAffine, RandomTranslation, TransformList
+from meddlr.forward.mri import SenseModel
+
+from meddlr.utils.events import get_event_storage
+
+
+class MotionModel2D:
```

Comment:
It seems like this class is just calling the function we wrote - what is the advantage of having a class here? There doesn't seem to be any state we are preserving.

If there is no state, just add the function to your eval script for now and call that in the MotionDataTransform

Reply:
So, I'm taking the function that we wrote and I'm writing it within the MotionDataTransform. I'm erasing this class as a result. Is that correct or was I supposed to do something else? Thanks!

Reply:
@ad12

Reply:
Correct - remove the class. Note, if the function depends on instance attributes in `MotionDataTransform`, make it an instance method. If it does not, make it a standalone function